### DISCUSSION TITLE
Anything like z-index to make sure some windows are always above others?

### DISCUSSION THREAD

**Discussion by laur89** [i3/i3#5094] [Q&A] [answered]
I'd like to be able to see some floating windows even if another window is currently showing in full-screen. Example: pulling up a clipboard manager to search for a recent value to use.

Is this currently possible in i3? If not, would the project consider a new feature; guess it's essentially a z-index-like feature?

**Comment by Airblader** [comment, marked as answer]
No, this isn't possible in i3.

There is "always on top" in the specifications, but we don't implement that. Even if we did, we probably we wouldn't respect that for fullscreen windows.

**Reply by laur89** [reply]
Just to be clear - this is nothing i3 project would consider?

**Reply by Airblader** [reply]
"Always on top" would be #4265 / #4478. If you want to make a case for why this should be supported when floating windows should be the exception, please open an issue. But at least the use case you mentioned here wouldn't be solved by that since always on top does not overrule fullscreen.

If you mean a proprietary z-index system, then no, we wouldn't want to add something like that.

**Reply by Airblader** [reply]
You might be interested in #2107 but that hasn't been implemented.

**Reply by laur89** [reply]
> You might be interested in #2107 but that hasn't been implemented.

Already following that one. That _would_ work for me, as with that option I'd be using fullscreen only for video, presentations and the like. Guess that's the answer to my problem.
